src/components/link/link.test.js

```diff
@@ -239,5 +225,95 @@ context("Test for Link component", () => {
           expect(callback).to.have.been.calledOnce;
         });
     });
+
+    describe("should render Link component and check accessibility", () => {
+      it("should check accessibility for link component", () => {
+        CypressMountWithProviders(<LinkComponent />);
+
+        cy.checkAccessibility();
+      });
+
+      // FE-4647
+      describe.skip("skip", () => {
+        it("should render Link disabled and check accessibility", () => {
+          CypressMountWithProviders(<LinkComponent disabled />);
+
+          cy.checkAccessibility();
+        });
+      });
+
+      it("should render Link with icon and check accessibility", () => {
+        CypressMountWithProviders(<LinkComponent icon="add" />);
+
+        cy.checkAccessibility();
+      });
+
+      it.each(["left", "right"])(
+        "should render Link with iconAlign prop set to %s and check accessibility",
+        (iconAlign) => {
+          CypressMountWithProviders(
+            <LinkComponent icon="add" iconAlign={iconAlign} />
+          );
+
+          cy.checkAccessibility();
+        }
+      );
+
+      it("should render Link with href prop and check accessibility", () => {
+        CypressMountWithProviders(<LinkComponent href={testCypress} />);
+
+        cy.checkAccessibility();
+      });
+
+      it.each(["top", "bottom", "left", "right"])(
+        "should render Link with tooltipPosition prop set to %s and check accessibility",
+        (tooltipPosition) => {
+          CypressMountWithProviders(
+            <Box m="250px">
+              <LinkComponent
+                icon="add"
+                tooltipMessage={testCypress}
+                tooltipPosition={tooltipPosition}
+              />
+            </Box>
+          );
+
+          cy.checkAccessibility();
+        }
+      );
+
+      it.each(["_blank", "_self", "_parent", "_top"])(
+        "should render Link with target prop set to %s and check accessibility",
+        (target) => {
+          CypressMountWithProviders(<LinkComponent target={target} />);
+
+          cy.checkAccessibility();
+        }
+      );
+
+      it("should render Link with ariaLabel prop and check accessibility", () => {
+        CypressMountWithProviders(<LinkComponent ariaLabel={testCypress} />);
+
+        cy.checkAccessibility();
+      });
+
+      it.each(["noopener", "noreferrer", "opener"])(
+        "should render Link with rel prop set to %s and check accessibility",
+        (rel) => {
+          CypressMountWithProviders(<LinkComponent rel={rel} />);
+
+          cy.checkAccessibility();
+        }
+      );
+
+      it.each(["default", "negative", "neutral"])(
+        "should render Link with variant prop set to %s and check accessibility",
+        (variant) => {
+          CypressMountWithProviders(<LinkComponent variant={variant} />);
+
+          cy.checkAccessibility();
+        }
+      );
```

Comment:
You could add a test for the 'OnADarkBackground' story from link.stories.tsx (the one I mentioned on the call).

Reply:
Thanks for the review, I added the test in, but there is an accessibility violation for this 'OnADarkground' which the ticket had been created in our backlog, so I 'skip' that test for now.